news-bundle/src/EventListener/NewsFeedListener.php

```diff
@@ -182,6 +182,10 @@ protected function getEnclosures(NewsModel $article, ItemInterface $item, Transf
         while ($files->next()) {
             $file = new File($files->path);
 
+            if (!$file->exists()) {
+                continue;
+            }
```

Comment:
If the file does not exist for whatever reason there would currently be an error.